docs/alerts.md

```diff
@@ -51,15 +129,91 @@ Elasticsearch will attempt to relocate shards away from a node for [reaching the
 
 ### Troubleshooting
 
-Check the disk space of the node.
-
+1. Identify the node on which elasticsearch is deployed:
+    ```
+   oc -n openshift-logging get po -o wide
+   ```
+
+2. Check the disk space on each node:
+   ```
+   oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_nodes/stats/fs?pretty
+   ```
+
+3. Check if the cluster is rebalancing:
+   ```
+   oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_cluster/health?pretty | grep relocating_shards
+   ```
+
+4. If there are any relocating shards present then:
+    - The High Watermark which is 90% by default has been exceeded.
+    - The shards will start relocating to a different node which has low disk usage and hasn’t crossed any watermark threshold limits.
+    - To allocate shards to this particular node, free up some spaces:
+        - Increase disk space on all nodes.
+        - If increasing disk space isn’t possible, then add a new data node to the cluster.
+        - If adding a new data node is problematic, then reduce the number of replicas
+            - Preferably on older indices.
+            - Get the index settings:
+              ```
+              oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=<elasticsearch_index_name>/_settings?pretty
+              ```
+            - Identify the number of replicas from the output of the above command.
+            - Lower the number of replicas if possible:
+              ```
+              oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=<elasticsearch_index_name>/_settings?pretty -X PUT -d '{"index.number_of_replicas":<number_of_replicas>}'
+              ```
+        - If nothing above helps, then last solution is to delete old indices
+            - Check the status of all indices on elasticsearch:
+              ```
+              oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- indices
+              ```
+            - Identify the index that can be deleted.
+            - Delete the index:
+              ```
+              oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=<elasticsearch_index_name> -X DELETE
+              ```
+    
```

Comment:
i think we also want to add a step to unlock all the indices now that the watermark level is below the threshold. ES will lock the indices automatically but it will not unlock them for ES6

Reply:
@ewolinetz what I had found is that ES will lock the indices only on reaching flood watermark level and not on low or high. That's why I have mentioned a step to unlock the indices in flood watermark troubleshooting.